components/stack/stack_constants.js

```diff
@@ -0,0 +1,19 @@
+export const DT_STACK_DIR = {
+  default: 'column',
+  column: 'column',
+  row: 'row',
+  'row-reverse': 'row-reverse',
+  'column-reverse': 'column-reverse',
+};
+
+export const DT_STACK_RESPONSIVE_BREAKPOINTS = ['sm', 'md', 'lg', 'xl'];
+
+export const DT_STACK_GAP = {
+  default: '0',
+  100: '0.1rem',
+  200: '0.2rem',
+  300: '0.4rem',
+  400: '0.8rem',
+  500: '1.6rem',
+  600: '3.2rem',
```

Comment:
Use the actual CSS vars here ex `var(--space-300)`

Reply:
With the style refactor we are not using anymore this styles, we just use it for validation